src/main/resources/languages/pl-PL.json

```diff
@@ -48,7 +48,8 @@
       "run_mode_help": "Tryb pracy serwera musi być ustawiony na 'HYBRID', 'DISPATCH_ONLY', lub 'GAME_ONLY'. Nie można wystartować Grasscutter...",
       "create_resources": "Tworzenie folderu resources...",
       "resources_error": "Umieść kopię 'BinOutput' i 'ExcelBinOutput' w folderze resources.",
-      "version": "Grasscutter versión: %s-%s"
+      "version": "Grasscutter versión: %s-%s",
+      "game_version": "Game versión: 2.7.0"
```

Comment:
Please change these so the version number is a variable.

Reply:
Done!